dmn-compatibility/public/index.html

```diff
@@ -1,10 +1,12 @@
 <!doctype html>
 <html lang="en">
 <head>
+  <!-- required viewer styles -->
   <link rel="stylesheet" href="../node_modules/dmn-js/dist/assets/diagram-js.css">
   <link rel="stylesheet" href="../node_modules/dmn-js/dist/assets/dmn-js-drd.css">
   <link rel="stylesheet" href="../node_modules/dmn-js/dist/assets/dmn-js-decision-table.css">
   <link rel="stylesheet" href="../node_modules/dmn-js/dist/assets/dmn-js-literal-expression.css">
+  <link rel="stylesheet" href="../node_modules/dmn-js/dist/assets/dmn-js-shared.css">

```

Comment:
Is this new? Or don't we use it?

Reply:
I think it was forgotten / or not needed in the very old setup (because the overlay icons were not shown)